app/serializers/group_serializer.rb

```diff
@@ -6,6 +6,7 @@ class GroupSerializer < ActiveModel::Serializer
              :key,
              :name,
              :created_at,
+             :creator_id,
```

Comment:
Note for another time: I think we actually want this to be something like :account_owner or :org_owner or something. Group creator can leave the group, but there should always be an org owner.